Library/ViewModels/ChangeEmailViewModelTests.swift

```diff
@@ -27,6 +28,21 @@ final class ChangeEmailViewModelTests: TestCase {
   private let warningMessageLabelHidden = TestObserver<Bool, Never>()
   private let verificationEmailButtonTitle = TestObserver<String, Never>()
 
+  // MARK: Computed Properties
+
+  private var userChangeEmailSuccessMockService: MockService {
+    guard let envelope = UserEnvelope<GraphUser>.userEnvelope(from: fetchUserQueryData) else {
+      return MockService()
```

Comment:
Should this be MockService() or should it pass a failure as the result?

Reply:
Well, I'm not sure that we can then pass the failure to the `withEnvironment(apiService:` parameter, might be simpler to just pass in the basic `MockService()` with no results to fetch, the test should fail in this anyway.